configs/modules/core/estimation/mview_mperson_topdown_estimator.py

```diff
@@ -0,0 +1,229 @@
+            type='BodyPosePriorHandler',
+            prior_loss=dict(
+                type='MaxMixturePriorLoss',
+                prior_folder='xrmocap_data',
+                num_gaussians=8,
+                loss_weight=4.78**2,
+                reduction='sum'),
+            logger=logger),
+        dict(
+            handler_key='pose_reg',
+            type='BodyPosePriorHandler',
+            prior_loss=dict(
+                type='PoseRegLoss', loss_weight=0.001, reduction='mean'),
+            logger=logger),
+        dict(
+            handler_key='keypoints3d_limb_len',
+            type='Keypoint3dLimbLenHandler',
+            loss=dict(
+                type='LimbLengthLoss',
+                convention='smpl',
+                loss_weight=1.0,
+                reduction='mean'),
+            logger=logger),
+    ],
+    stages=[
+        # stage 0
+        dict(
+            n_iter=10,
+            ftol=1e-4,
+            fit_global_orient=False,
+            fit_transl=False,
+            fit_body_pose=False,
+            fit_betas=True,
+            keypoints3d_mse_weight=0.0,
+            keypoints3d_limb_len_weight=1.0,
+            shape_prior_weight=5e-3,
+            joint_prior_weight=0.0,
+            pose_reg_weight=0.0,
+            pose_prior_weight=0.0),
+        # stage 1
+        dict(
+            n_iter=50,
+            ftol=1e-4,
+            fit_global_orient=True,
+            fit_transl=True,
+            fit_body_pose=False,
+            fit_betas=False,
+            keypoints3d_mse_weight=1.0,
+            keypoints3d_limb_len_weight=0.0,
+            shape_prior_weight=0.0,
+            joint_prior_weight=0.0,
+            pose_reg_weight=0.0,
+            pose_prior_weight=0.0,
+            body_weight=5.0,
+            use_shoulder_hip_only=True),
+        # stage 2
+        dict(
+            n_iter=120,
+            ftol=1e-4,
+            fit_global_orient=True,
+            fit_transl=True,
+            fit_body_pose=True,
+            fit_betas=False,
+            keypoints3d_mse_weight=10.0,
+            keypoints3d_limb_len_weight=0.0,
+            shape_prior_weight=0.0,
+            joint_prior_weight=0.0,
+            pose_reg_weight=0.001,
+            pose_prior_weight=0.0,
+            body_weight=1.0,
+            use_shoulder_hip_only=False),
+    ],
+)
+
+triangulator = dict(
+    type='AniposelibTriangulator',
+    camera_parameters=[],
+    logger=logger,
+)
+point_selectors = [
+    dict(
+        type='HybridKps2dSelector',
+        triangulator=dict(
+            type='AniposelibTriangulator', camera_parameters=[],
+            logger=logger),
+        verbose=verbose,
+        ignore_kps_name=['left_eye', 'right_eye', 'left_ear', 'right_ear'],
+        convention=pred_kps3d_convention,
+    )
+]
+
+kps3d_optimizers = [
+    dict(type='TrajectoryOptimizer', verbose=verbose, logger=logger),
+    dict(type='NanInterpolation', verbose=verbose, logger=logger),
+    # dict(
```

Comment:
What about keep this `SMPLShapeAwareOptimizer` in the config under `configs/modules`, and remove `SMPLShapeAwareOptimizer` from the config under `configs/MVPose`?

Reply:
OK

Reply:
Do we need `kps3d_optimizers` in MVPose?

Reply:
In MVPose dir, we offer the best setting of kps3d_optimizers, it could be empty, or a sequence.